package.json

```diff
@@ -28,6 +28,7 @@
         "@nestjs/core": "^9.0.0",
         "@nestjs/microservices": "^9.1.4",
         "@nestjs/platform-express": "^9.0.0",
+        "@origranot/ts-logger": "^1.12.0",
```

Comment:
We want to keep the client as lightweight as possible so we avoid from adding packages we can implement by ourselves.